t/errlog.t

```diff
@@ -882,3 +882,53 @@ log body:\d{4}/\d{2}/\d{2} \d{2}:\d{2}:\d{2} \[error\] (\d+).*access_by_lua\(ngi
 new line, client: 127.0.0.1, server: localhost, request: "GET /t HTTP/1.1", host: "localhost"
 \z
 --- skip_nginx: 2: <1.11.2
+
+
+
+=== TEST 22: user-supplied Lua table to hold the result(with no error log)
```

Comment:
@membphis I'm afraid this test title does not state the test case's intention accurately. Maybe we should be more specific here?